worlds/stardew_valley/__init__.py

```diff
@@ -126,7 +126,7 @@ def create_items(self):
 
         locations_count = len([location
                                for location in self.multiworld.get_locations(self.player)
-                               if not location.event])
+                               if not location.advancement])
```

Comment:
I think this change may not be correct (and the same in tests). I think the `.event` was used here to decide which are event locations before fill happened, so instead this should probably check `not .address`?